.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+> Please fill out this template when filing an issue.
```

Comment:
@bmunkholm talked about adding an introduction as well yesterday. Perhaps rephrase the first sentence to something like "In order for us to help you most effectively, please fill out this template when filing an issue."